9.2/root/usr/bin/hooks.d/post-init

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
The environment variables with DB name and access are available (set) at the time `post-init` script is running, so yes, user can either use them or connect to the daemon using `psql` like it is done in `set_passwords` function.

One very simple use case:
As ISP dev-ops engineer, I want to have own image based on `centos/postgresql-95-centos7`, so every initialized database will have Europe time-zone set.
Such engineer will create the following post-init file:
```
psql --command "SET TIME ZONE 'Europe/Rome'";
```
and the following Dockerfile:
```
FROM centos/postgresql-95-centos7
USER 0
COPY post-init /usr/bin/hooks.d/
USER 26
```

Another use case:
As ISP dev-ops engineer, I want to have own image based on `centos/postgresql-95-centos7`, but create not only access for normal use cases, but also separate user for monitoring, so monitoring tool has only limited permissions.
Such engineer will create the following post-init file:
```
if [ "${NEED_TO_CREATE_USERS:-}" == "yes" ]; then
  createuser "$POSTGRESQL_MONITORING_USER"
fi

if [ -v POSTGRESQL_MONITORING_PASSWORD ]; then
  psql --command "ALTER USER \"${POSTGRESQL_MONITORING_USER}\" WITH ENCRYPTED PASSWORD '${POSTGRESQL_MONITORING_PASSWORD}';"
fi
```
and the same Dockerfile as in the example above.